src/Model/Windowing.py

```diff
@@ -19,26 +19,18 @@ def windowing_model(text, init):
     # Use custom window/level for manual fusion overlays (init[3])
     #TODO This is just a patch. Need to figure out why VTK doesnt work with default dicom windowing
     if init[3]:
-        # You can use a custom dictionary here, or load from MovingDictContainer
-        moving_dict_container = MovingDictContainer()
-        dict_windowing_moving = moving_dict_container.get("dict_windowing_moving")
-        if dict_windowing_moving and text in dict_windowing_moving:
-            windowing_limits = dict_windowing_moving[text]
-        else:
-            # Fallback to a hardcoded set or the fixed image's presets
-            custom_presets = {
-                "Normal": [390, 40],  # Soft tissue, covers -160 to 240 HU
-                "Lung": [1600, -600],  # Lung, covers -1400 to 200 HU
-                "Bone": [2000, 300],  # Bone, covers -700 to 1300 HU
-                "Brain": [80, 40],  # Brain, covers 0 to 80 HU
-                "Soft Tissue": [440, 40],  # Alias for Normal
-                "Head and Neck": [275, 40],  # Covers -147.5 to 227.5 HU
-            }
-            windowing_limits = custom_presets.get(text, [400, 40])
+        custom_presets = {
```

Comment:
This is essentially a fixed dictionary, unless it's being updated elsewhere.
And for these hardcoded settings, it is a fixed dictionary:
WINDOW_LEVEL_PRESETS = { "LUNG": [1600, -600], ...}
DEFAULT_PRESET = WINDOW_LEVEL_PRESETS["NORMAL"]
And then just do the lookup
windowing_limits = WINDOW_LEVEL_PRESETS.get(text, DEFAULT_PRESET)